Classes/StorageFactory.php

```diff
@@ -28,6 +28,12 @@ class StorageFactory
     protected $credentialProfiles;
 
     /**
+     * @Flow\Inject
```

Comment:
As highlighted by the red background, something is wrong here. Hard to tell what exactly happens, but there is non-standard whitespace characters involved. Check using a hex editor… BTW, PhpStorm also complains here:

<img width="379" alt="weird-whitespace" src="https://cloud.githubusercontent.com/assets/95873/15408017/50866816-1e0e-11e6-82e7-245826a57b13.png">